python/tk_multi_loader/dialog.py

```diff
@@ -676,12 +677,17 @@ def _on_home_clicked(self):
                     if item is not None:
                         # find an absolute match! Break the search.
                         found_item = item
+                        found_model = model
                         break
 
         if found_preset is None:
             # no suitable item found. Use the first tab
             found_preset = self.ui.entity_preset_tabs.tabText(0)
 
+        if found_model and found_item:
```

Comment:
I'm a little late to the party but can you add some comments to explain why we have to reselect everything, + the ticket id